src/distances.jl

```diff
@@ -10,13 +12,13 @@ end
 
 
 """
-    haversine{T<:Real}(c1::Tuple{T,T}, c2::Tuple{T,T})
+    haversine(c1::Tuple{T,T}, c2::Tuple{T,T}) where T <: Real
```

Comment:
👍

Reply:
This is the same case as the one you raised in the `LoadBus`/`GenBus` case. It is the same syntax change.

Reply:
it's a good change! I misread the case with the constructor above, as you duly noticed 😆